### ISSUE TITLE
Hi, I have some questions about your method

### ISSUE THREAD

**Issue by mks0601** [1adrianb/human-pose-estimation#6]
Hi, thanks for sharing the code of your work.
I have some questions about your method.

Have you tried end-to-end learning (joint learning of detector and regression network)? If so, can you tell me the performance and hyper-parameters (e.g., learning rate)?

Also, how the performance change if deconv layer is initialized by random gaussian distribution( not initialized as bilinear upsampling)?
